### PR TITLE
chore(jsondata): add German L10n

### PR DESCRIPTION
**Pull request by caugner**
### Description

Adds German L10n translations for the [experimental German locale](https://github.com/orgs/mdn/discussions/741).

### Motivation

Some sidebars use these L10n keys, so I'm adding them here.

### CODE REVIEW

File: files/jsondata/L10n-CSS.json [hunk 1]
```diff
@@ -66,6 +72,7 @@
     "zh-TW": "選擇器"
   },
   "Basic Selectors": {
+    "de": "Basic-Selektoren",
```
Comment (argl) [inline files/jsondata/L10n-CSS.json:75]: consider "Basis-Selektoren"?

File: files/jsondata/L10n-Common.json [hunk 2]
```diff
@@ -239,6 +261,7 @@
     "ru": "раздел"
   },
   "summary": {
+    "de": "Die Dokumentation zu diesem Thema wurde noch nicht verfasst; hilf uns dabei!",
```
Comment (argl) [inline files/jsondata/L10n-Common.json:264]: consider "Die Dokumentation zu diesem Thema wurde noch nicht verfasst. Hilf uns dabei!"

File: files/jsondata/L10n-SVG.json [hunk 3]
```diff
@@ -1,5 +1,6 @@
 {
   "animationAdditionAttributes": {
+    "de": "Attribute für Animationserweiterungen",
```
Comment (argl) [inline files/jsondata/L10n-SVG.json:3]: consider "Attribute der Animationserweiterungen", to be in consistent with the "Zielattribute" below

File: files/jsondata/L10n-SVG.json [hunk 4]
```diff
@@ -88,6 +96,7 @@
     "zh-TW": "任意元素或字元資料"
   },
   "anyNumberOfElementsAnyOrder": {
+    "de": "Beliebige Anzahl der folgenden Elemente in beliebiger Reihenfolge:",
```
Comment (argl) [inline files/jsondata/L10n-SVG.json:99]: consider "Beliebige Anzahl folgender Elemente, in beliebiger Reihenfolge:"

File: files/jsondata/L10n-SVG.json [hunk 5]
```diff
@@ -154,6 +168,7 @@
     "zh-TW": "類型"
   },
   "characterDataElementsInAnyOrder": {
+    "de": "Zeichendaten und beliebige Anzahl der folgenden Elemente, in beliebiger Reihenfolge:",
```
Comment (argl) [inline files/jsondata/L10n-SVG.json:171]: consider "Zeichendaten und beliebige Anzahl folgender Elemente, in beliebiger Reihenfolge:"

File: files/jsondata/L10n-SVG.json [hunk 6]
```diff
@@ -253,6 +276,7 @@
     "zh-TW": "濾鏡属性"
   },
   "filterPrimitiveElement": {
+    "de": "Primitive Filterelemente",
```
Comment (argl) [inline files/jsondata/L10n-SVG.json:279]: Does this describe a single element? Like "Primitives Filterelement"?

**Review by argl — APPROVED** [review]
A few nits, otherwise great!

**Review by bsmth — APPROVED** [review]
Andi's already got the DE aspect, so leaving a +1 from a structural perspective

File: files/jsondata/L10n-SVG.json (hunk 6 above)
Comment (caugner) [reply files/jsondata/L10n-SVG.json]: Fixed in 77d66d05a0a630be5ee3b4ebdc371806da92760e.

File: files/jsondata/L10n-SVG.json (hunk 5 above)
Comment (caugner) [reply files/jsondata/L10n-SVG.json]: Fixed in 77d66d05a0a630be5ee3b4ebdc371806da92760e.

File: files/jsondata/L10n-SVG.json (hunk 4 above)
Comment (caugner) [reply files/jsondata/L10n-SVG.json]: Fixed in 77d66d05a0a630be5ee3b4ebdc371806da92760e.

File: files/jsondata/L10n-SVG.json (hunk 3 above)
Comment (caugner) [reply files/jsondata/L10n-SVG.json:3]: Fixed in 77d66d05a0a630be5ee3b4ebdc371806da92760e.

File: files/jsondata/L10n-CSS.json (hunk 1 above)
Comment (caugner) [reply files/jsondata/L10n-CSS.json]: Fixed in 77d66d05a0a630be5ee3b4ebdc371806da92760e.

File: files/jsondata/L10n-Common.json (hunk 2 above)
Comment (caugner) [reply files/jsondata/L10n-Common.json]: Fixed in 77d66d05a0a630be5ee3b4ebdc371806da92760e.

**Pull request merged**